### PR TITLE
homepage charts testing

### PR DESCRIPTION
**Pull request by Andrew-Sofos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "dotenv": "^8.2.0",
     "ethereum-blockies-base64": "^1.0.2",
     "framer-motion": "^4.1.3",
-    "gatsby": "^3.0.4",
+    "gatsby": "^3.7.2",
```
Comment (samajammin) [inline package.json:24]: Is there a reason why we're updating Gatsby here? Seems that could/should be done in a separate PR to isolate any potential issues.

Comment (Andrew-Sofos) [reply package.json:24]: Hi @samajammin , I didn't do it manually I think the action: yarn add recharts, updated the gatsby in the prosses of installing the library.
